lonestar/analytics/cpu/clustering/leidenClustering.cpp

```diff
@@ -316,20 +315,21 @@ void runMultiPhaseLouvainAlgorithm(Graph& graph, uint64_t min_graph_size,
 
 int main(int argc, char** argv) {
   galois::SharedMemSys G;
-  LonestarStart(argc, argv, name, desc, url);
+  LonestarStart(argc, argv, name, desc, url, inputFile.c_str());
 
-  Graph graph, graph_next;
-  Graph* graph_curr;
+  galois::StatTimer totalTime("TimerTotal");
+  totalTime.start();
```

Comment:
Stop missing.

See other comment for whether or not this is being done on purpose :p

I'm just marking them here in case it wasn't. (though I might miss some of them due to human error)